### ISSUE TITLE
Ubuntu 21.04 Hirsute package

### ISSUE THREAD

**Issue by srh** [rethinkdb/rethinkdb#6959]
We should make a 21.04 Hirsute package for amd64.  On Hirsute, RethinkDB actually doesn't build out of the box -- an error occurs when building v8.

A Groovy package wouldn't hurt either, though it only has 3 months remaining, and that build works without needing any changes to RethinkDB.

We might need to upgrade v8.

**Comment by dettus** [comment]
I agree. Maybe this helps:


```
gdb --args "/home/det/Downloads/compile/rethinkdb-2.4.1/build/external/v8_3.30.33.16-patched/build/out/x64.release/mksnapshot" --log-snapshot-positions --logfile "/home/det/Downloads/compile/rethinkdb-2.4.1/build/external/v8_3.30.33.16-patched/build/out/x64.release/obj.target/v8_snapshot/geni/snapshot.log" --random-seed 314159265 "/home/det/Downloads/compile/rethinkdb-2.4.1/build/external/v8_3.30.33.16-patched/build/out/x64.release/obj.target/v8_snapshot/geni/snapshot.cc"
GNU gdb (Ubuntu 10.1-2ubuntu2) 10.1.90.20210411-git
Copyright (C) 2021 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<https://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from /home/det/Downloads/compile/rethinkdb-2.4.1/build/external/v8_3.30.33.16-patched/build/out/x64.release/mksnapshot...
(No debugging symbols found in /home/det/Downloads/compile/rethinkdb-2.4.1/build/external/v8_3.30.33.16-patched/build/out/x64.release/mksnapshot)
(gdb) run
Starting program: /home/det/Downloads/compile/rethinkdb-2.4.1/build/external/v8_3.30.33.16-patched/build/out/x64.release/mksnapshot --log-snapshot-positions --logfile /home/det/Downloads/compile/rethinkdb-2.4.1/build/external/v8_3.30.33.16-patched/build/out/x64.release/obj.target/v8_snapshot/geni/snapshot.log --random-seed 314159265 /home/det/Downloads/compile/rethinkdb-2.4.1/build/external/v8_3.30.33.16-patched/build/out/x64.release/obj.target/v8_snapshot/geni/snapshot.cc
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7ffff7a14640 (LWP 266758)]
[New Thread 0x7ffff7213640 (LWP 266759)]
[New Thread 0x7ffff6a12640 (LWP 266760)]
[New Thread 0x7ffff6211640 (LWP 266761)]
[New Thread 0x7ffff5a10640 (LWP 266762)]

Thread 1 "mksnapshot" received signal SIGSEGV, Segmentation fault.
0x00005555557295c2 in v8::internal::RootMarkingVisitor::VisitPointer(v8::internal::Object**) ()
(gdb) quit
A debugging session is active.

	Inferior 1 [process 266754] will be killed.

Quit anyway? (y or n) y
```

**Comment by srh** [comment]
Building the package is fixed in v2.4.x (with QuickJS replacing v8) and package generation is working.  There will be a package for active latest Ubuntus in v2.4.2 release.  (21.04, 21.10, and 22.04 are all working.)  See #6975 for updates.
